_posts/zh/newsletters/2021-04-07-newsletter.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Bitcoin Optech Newsletter #143'
+permalink: /zh/newsletters/2021/04/07/
+name: 2021-04-07-newsletter-zh
+slug: 2021-04-07-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 包含定期栏目，其中包括新版本发布与候选发布的公告以及对流行的比特币基础设施项目所做的值得注意的更改。
+
+## 新闻
+
+*本周无值得报道的新闻。*
+
+## 发布与候选发布
+
+- [C-Lightning 0.10.0][C-Lightning 0.10.0] 是该闪电网络节点软件的最新主要版本。它对其 API 进行了多项增强，并包括对[双向资助][topic dual funding] 的实验性支持。
+
+- [BTCPay 1.0.7.2][BTCPay 1.0.7.2] 修复了在上周安全发布后发现的一些小问题。
+
+## 值得注意的代码和文档更改
+
+*本周值得注意的更改发生在 [Bitcoin Core][bitcoin core repo]、[C-Lightning][c-lightning repo]、[Eclair][eclair repo]、[LND][lnd repo]、[Rust-Lightning][rust-lightning repo]、[libsecp256k1][libsecp256k1 repo]、[硬件钱包接口（HWI）][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay Server][btcpay server repo]、[比特币改进提案（BIPs）][bips repo]以及[闪电网络规范（BOLTs）][bolts repo]。*
+
+- [Bitcoin Core #20286][] 从 `gettxout`、`getrawtransaction`、`decoderawtransaction`、`decodescript`、`gettransaction` 这些 RPC 方法的响应中，以及从 `/rest/tx`、`/rest/getutxos`、`/rest/block` 这些 REST 端点中移除了 `addresses` 和 `reqSigs` 字段。当存在明确定义的地址时，响应中现在包括可选字段 `address`。这些已弃用的字段之前用于裸多重签名场景，而这在当今网络中已无实质用途。在 Bitcoin Core 23.0 移除该选项之前，这些已弃用的字段仍可通过配置选项 `-deprecatedrpc=addresses` 输出。
+
+- [Bitcoin Core #20197][] 通过更新入站节点剔除逻辑改善了节点连接的多样性，以保护运行时间最长的 onion 节点。它还为当前的剔除保护逻辑增加了单元测试覆盖率。由于 onion 节点的延迟通常高于 IPv4 和 IPv6 节点，它们在剔除条件下一直处于不利地位，导致用户提交了[多个][Bitcoin Core #11537][问题][Bitcoin Core #19500]。最初的应对措施[news114 core19670] 为本地主机节点预留了插槽，以此作为 onion 节点的代理。后来又添加了[对入站 onion 连接的显式检测][news118 core19991]。
```

Comment:
I think the [news114 core19670] link is missing link text

Reply:
Thank you! The missing link is corresponded to several zh characters. I fix it.